styles.css

```diff
@@ -95,6 +95,24 @@ input {
   width: 30%;
 }
 
+.search-box {
+  display: flex;
+}
+
+.search-icon {
+  height: 39px;
```

Comment:
Should we change a lot of these px's to percentages so that it is more dynamic when we change the size of the browser?

Reply:
I was getting weird stuff w percentages and had to work w either end or px. I agree that it would be much better w percentages for diff screens!